app/src/main/res/values/strings.xml

```diff
@@ -657,6 +657,14 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_bicycle_parking_type_building">Building</string>
     <string name="quest_bicycle_parking_type_handlebarholder">Handlebar holder</string>
 
+    <string name="quest_bicycle_rental_type_title">What type of bicycle rental is this?</string>
+    <string name="quest_bicycle_rental_type_docking_station">Docking station (locks bikes)</string>
+    <string name="quest_bicycle_rental_type_dropoff_point">Designated spot (bikes locked individually)</string>
```

Comment:
I don't really have a better suggestion right now, but it seems a bit odd that the comment in brackets for both could equally apply to the other one.

```suggestion
    <string name="quest_bicycle_rental_type_docking_station">Docking station (locks bikes in place)</string>
    <string name="quest_bicycle_rental_type_dropoff_point">Designated spot (bikes locked individually)</string>
```

Reply:
You're right, that bugged me too. Maybe like this?

```suggestion
    <string name="quest_bicycle_rental_type_docking_station">Docking station which locks bikes</string>
    <string name="quest_bicycle_rental_type_dropoff_point">Designated spot (bikes locked individually)</string>
```

Reply:
Yes, that feels better to me, I think I pondered something similar but then tried to match your brackets!